src/views/faq/faq.jsx

```diff
@@ -0,0 +1,421 @@
+var render = require('../../lib/render.jsx');
+
+var Page = require('../../components/page/www/page.jsx');
+var FlexRow = require('../../components/flex-row/flex-row.jsx');
+var SubNavigation = require('../../components/subnavigation/subnavigation.jsx');
+var TitleBanner = require('../../components/title-banner/title-banner.jsx');
+
+require('./faq.scss');
+
+var Faq = React.createClass({
+    type: 'Faq',
+    render: function () {
+        return (
+            <div className="faq">
+                <TitleBanner className="masthead">
+                    <div className="inner">
+                        <h1>FAQ</h1>
+                        <p className="intro"><FormattedMessage id='faq.intro' /></p>
+                    </div>
+                    <div className="band">
+                        <SubNavigation className="inner">
+                            <a href="#about-scratch">
+                                <li>
+                                    <FormattedMessage id='faq.aboutTitle' />
+                                </li>
+                            </a>
+                            <a href="#privacy">
+                                <li>
+                                    <FormattedMessage id='faq.privacyTitle' />
+                                </li>
+                            </a>
+                            <a href="#remix">
+                                <li>
+                                    <FormattedMessage id='faq.remixTitle' />
+                                </li>
+                            </a>
+                            <a href="#accounts">
+                                <li>
+                                    <FormattedMessage id='faq.accountsTitle' />
+                                </li>
+                            </a>
+                            <a href="#permissions">
+                                <li>
+                                    <FormattedMessage id='faq.permissionsTitle' />
+                                </li>
+                            </a>
+                            <a href="#inappropriate-content">
+                                <li>
+                                    <FormattedMessage id='faq.inappropriateContentTitle' />
+                                </li>
+                            </a>
+                            <a href="#clouddata">
+                                <li>
+                                    <FormattedMessage id='faq.cloudDataTitle' />
+                                </li>
+                            </a>
+                        </SubNavigation>
+                    </div>
+                </TitleBanner>
+                <div className="inner">
+                    <section id="about-scratch">
+                        <span className="nav-spacer"></span>
+                        <h2><FormattedMessage id='faq.aboutTitle' /></h2>
+                        <FlexRow className="sidebar-row">
```

Comment:
I think that these might make more sense if they use the `dl/dd/dt` tag structure, since they have sort of "title" text (the question), and then "description" text (the answer). Also, since they're always going to be in columns, I think we could take out the use of `FlexRow`